test/plugins/features/movies/controller.test.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const Controller = require('../../../../lib/plugins/features/movies/controller');
+const Movie = require('../../../../lib/models/movie');
+
+describe('movie controller', () => {
+
+  describe('create', () => {
+
+    it ('creates a movie', () => {
+      const payload = { title: 'WALL-E'};
+
+      return Controller.create(payload).then((movie) => {
+        expect(movie.get('title')).to.eql(payload.title);
+
+        return new Movie({ id: movie.id}).fetch();
+      })
+      .then((movie) => {
+        expect(movie.get('title')).to.eql(payload.title);
+      });
+    });
+
+  });
+});
```

Comment:
Nit: Generally we leave a newline after the first line and before the last line of a `describe` block, similar to how you arranged the `create` block